implementations/rust/ockam/ockam_command/tests/bats/projects_orchestrator.bats

```diff
@@ -74,26 +84,25 @@ teardown() {
   # Change to a new home directory where there are no enrolled identities
   setup_home_dir
   NON_ENROLLED_OCKAM_HOME=$OCKAM_HOME
-  cp -r $ENROLLED_OCKAM_HOME/. $NON_ENROLLED_OCKAM_HOME
+  "$OCKAM" project import --project-file $PROJECT_PATH
 
   # Create a named default identity
   run_success "$OCKAM" identity create green
   green_identifier=$($OCKAM identity show green)
 
   # Create node for the non-enrolled identity using the exported project information
-  run_success "$OCKAM" node create green --project $PROJECT_NAME
+  run_success "$OCKAM" node create green
 
   # Node can't create relay as it isn't a member
   fwd=$(random_str)
-  run_failure "$OCKAM" relay create --identity green "$fwd"
+  run_failure "$OCKAM" relay create "$fwd"
 
   # Add node as a member
   OCKAM_HOME=$ENROLLED_OCKAM_HOME
-  run_success "$OCKAM" project ticket --member "$green_identifier" --attribute role=member
+  run_success "$OCKAM" project ticket --member "$green_identifier" --attribute role=member --relay $fwd
 
   # The node can now access the project's services
   OCKAM_HOME=$NON_ENROLLED_OCKAM_HOME
-  fwd=$(random_str)
   run_success "$OCKAM" relay create "$fwd"
```

Comment:
> it uses the "green" node created at line 94?

Yes because this is the default node.

> run_failure "$OCKAM" relay create --identity green "$fwd"

That failed simply because `--identity` is not an option for `relay create`. 

> run_success "$OCKAM" relay create "$fwd" worked because it was using the default identity, that is the one that have * permissions on relays. So it worked with nothing to do with the $green_identifier being enrolled.

Yes, in that case, the default identity was the first one created in the database that was copied at the beginning of the setup. And it was the initially enrolled identity that has a `*` permission indeed!

Reply:
I have added some comments to explain what's going on with the green node and the green identity.